code/src/App.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react'
+import React from "react";
+import { Provider } from "react-redux";
+import { combineReducers, createStore, compose } from "@reduxjs/toolkit";
+import persistState from 'redux-localstorage';
+
+import { Container, Footer } from "./styled-components";
+
+import StartPage from "components/StartPage";
+
+import tasks from "reducers/tasks";
+
+const enhancer = compose(persistState());
```

Comment:
Hmm I don't really know what the "compose" means...I have to read up on that as well :-)